utilities/rpp-performancetests/HIP_NEW/BatchPD_hip_pkd3.cpp

```diff
@@ -34,6 +34,27 @@ void check_hip_error(void)
     }
 }
 
+void swap (unsigned int *a, unsigned int *b)
```

Comment:
Please revert all changes from this file. Lets not touch the HIP performance tests file in this PR.